napari/layers/labels/labels.py

```diff
@@ -127,6 +138,22 @@ def __init__(
         self._num_colors = num_colors
         colormap = ('random', colormaps.label_colormap(self.num_colors))
 
+        if properties is None:
+            self._properties = {}
+            self._property_choices = {}
```

Comment:
Do we need to create `_property_choices` here?

I guess might use it in the future when we start thinking about adding properties as labels get added, but for now I don't think it is needed / doesn't appear in the other arm of the `else`

Reply:
>Do we need to create _property_choices here?

No, we don't. Have removed it!